packages/stylelint-config/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@readme/stylelint-config",
+  "version": "1.0.0",
+  "description": "ReadMe coding standards for styles",
+  "main": "index.js",
+  "files": [
+    "index.js"
+  ],
+  "scripts": {
+    "lint": "eslint ."
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/readmeio/stylelint-config.git"
+  },
+  "author": "Engineers <engineers@readme.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/readmeio/stylelint-config/issues"
+  },
+  "homepage": "https://github.com/readmeio/stylelint-config#readme",
+  "dependencies": {
+    "postcss": "^8.4.12",
+    "stylelint": "^14.6.0",
+    "stylelint-config-prettier": "^9.0.3",
+    "stylelint-config-sass-guidelines": "^9.0.1",
+    "stylelint-config-standard": "^25.0.0",
+    "stylelint-config-standard-scss": "^3.0.0",
+    "stylelint-prettier": "^2.0.0"
+  },
+  "devDependencies": {
+    "@readme/eslint-config": "^8.5.1"
```

Comment:
Can remove this and re-install it with `npm install ../eslint-config` so it'll get picked up from the monorepo.

Reply:
Interestingly, this should work the same b/c lerna replaces all local packages in its node_modules dir with symlinks
<img width="850" alt="image" src="https://user-images.githubusercontent.com/751279/160673792-297fc768-f4a2-45f1-b59d-576cc8a257d9.png">

However, I agree with your suggestion b/c npm won't have to bother bumping versions for locally linked packages. And it makes it clearer which packages are locally vs externally installed.

Will do!

Reply:
Agreed, it definitely makes the distinction of "this is a local package" clearer eventhough Lerna and NPM workspaces handles it mostly automatically.